.github/workflows/no-build-required.yml

```diff
@@ -0,0 +1,15 @@
+name: 'build-test'
+on:
+  pull_request:
+    paths-ignore:
+      - '.github/dependabot.yml'
+      - '.github/workflows/**'
+      - 'action.yml'
+      - 'package.json'
+      - 'package-lock.json'
+      - 'src/**'
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
ubuntu-latest requires a pull of about 300-400 mb, just to echo.

If we're already running on ubuntu-latest then it'll be cached on the runner, but otherwise that's a bunch of wasted time and maybe want to `run-on alpine` or something instead

Reply:
This isn't an image; it's the GitHub Actions runner (`ubuntu-latest`).